scripts/ci/lint_pull_request.sh

```diff
@@ -11,7 +11,7 @@ CHANGED_FILES=$(git diff --name-only --diff-filter=AM master..$CURRENT_BRANCH |
 
 if [[ -n "$CHANGED_FILES" ]]; then
   set -x
-  $USER_BASE/bin/gjslint --strict --jslint_error=all --exclude_files=deps.js $CHANGED_FILES
+  $USER_BASE/bin/gjslint --strict --jslint_error=all --exclude_files=deps.js,alltests.js $CHANGED_FILES
```

Comment:
You can revert this. Made this change in https://github.com/google/closure-library/commit/a785b1ffc7b71f78a7d2cbed8f7b48faa7934bb0.